uplink/ecclient/client.go

```diff
@@ -401,14 +401,14 @@ func (ec *ecClient) Delete(ctx context.Context, limits []*pb.AddressedOrderLimit
 				Address: addressedLimit.GetStorageNodeAddress(),
 			})
 			if err != nil {
-				ec.log.Sugar().Errorf("Failed dialing for deleting piece %s from node %s: %v", limit.PieceId, limit.StorageNodeId, err)
+				ec.log.Sugar().Debugf("Failed dialing for deleting piece %s from node %s: %v", limit.PieceId, limit.StorageNodeId, err)
```

Comment:
Since you're updating these lines anyway, can you remove the sugar logger? You could do something like this instead:
`ec.log.Debug("Failed dialing for deleting piece from node", zap.String("piece id", limit.PieceId.String()), zap.String("node id", limit.StorageNodeId.String()), zap.Error(err))`

Reply:
There are a lot of other logs in this file that should eventually be updated in the same way, but that doesn't need to be a part of this PR unless you want.

Reply:
okay I went ahead and removed the sugared logger from the file